Knoema.Client/Client.cs

```diff
@@ -121,7 +121,7 @@ private HttpClient GetApiClient(int timeout = HttpClientTimeout)
             return JsonConvert.DeserializeObject<T>(readString);
         }
 
-        private Task<T> ApiPost<T>(string path, object obj)
+        private Task<T> ApiPost<T>(string path, object obj, int timeout = HttpClientTimeout)
```

Comment:
I do not see how this timeout parameter passes to the private async Task<T> ApiPost<T>(string path, HttpContent content, int timeout = HttpClientTimeout)

Reply:
Fixed.